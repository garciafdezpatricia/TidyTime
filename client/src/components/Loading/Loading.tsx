

export default function Loader() {
    return (
        <div className="loading-page">
            <h2>TidyTime</h2>
            <div className="loading-bar-container">
                <div className="loading-bar"></div>
            </div>
        </div>
    )
}