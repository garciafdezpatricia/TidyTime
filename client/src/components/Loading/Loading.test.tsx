import { render } from "@testing-library/react";
import '@testing-library/jest-dom';
import Loading from '@/src/components/Loading/Loading';

describe('Loading', () => {
    it('renders without crashing', () => {
      const { container } = render(<Loading/>);
      expect(container).toBeInTheDocument();
    });

    it('matches snapshot', () => {
        const { asFragment } = render(<Loading/>);
        expect(asFragment()).toMatchSnapshot();
      });
  });