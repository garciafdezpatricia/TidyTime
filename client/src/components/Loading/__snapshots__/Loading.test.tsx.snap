// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="loading-page"
  >
    <h2>
      TidyTime
    </h2>
    <div
      class="loading-bar-container"
    >
      <div
        class="loading-bar"
      />
    </div>
  </div>
</DocumentFragment>
`;
